src/test/java/ut/org/catrobat/jira/timesheet/services/impl/TimesheetServiceImplTest.java

```diff
@@ -165,21 +164,20 @@ public void testUpdateTimesheet() {
         Timesheet.State stateOld = Timesheet.State.AUTO_INACTIVE;
 
         timesheet.setHoursCompleted(targetHoursCompletedOld);
-        timesheet.setHoursPracticeCompleted(targetHoursPracticeOld);
+        timesheet.setHoursCompleted(targetHoursPracticeOld);
         timesheet.setLatestEntryBeginDate(latestEntryDateOld);
         timesheet.setState(stateOld);
         timesheet.save();
 
         int targetHoursCompletedNew = 110;
-        int targetHoursPracticeNew = 170;
         Date latestEntryDateNew = new Date(200);
         Timesheet.State stateNew = Timesheet.State.ACTIVE;
 
         Timesheet updatedTimesheet = service.updateTimesheet(timesheet.getID(), targetHoursCompletedNew,
-                targetHoursPracticeNew, latestEntryDateNew, stateNew);
+                targetHoursCompletedNew, latestEntryDateNew, stateNew);
```

Comment:
Remove the deprecated `targetHoursPractice` parameter and argument at the method call.